.github/workflows/ci_trusty.yml

```diff
@@ -0,0 +1,47 @@
+name: CI - Ubuntu Trusty
+
+on:
+  # direct pushes to protected branches are not supported
+  pull_request:
+  # run every day, at 6am UTC
+  schedule:
+    - cron: '0 6 * * *'
```

Comment:
```suggestion
```
There will be no further `indigo` syncs that could break thing right?

Reply:
No public syncs no.